src/api/index.js

```diff
@@ -30,7 +30,7 @@ export default class Api {
      * @param {Array<Object>} config
      */
     constructor(config) {
-        this.NS_RULE = /^(.+)\.(.+)$/;
+        this.NS_RULE = /^(?:([^.]+)\.)?([^.]+)$/;
```

Comment:
```js
const matches = key.substr(namespace.length + 1).match(this.NS_RULE);
/*
key: 'devtools.inspectedWindow.tabId'
namespace: 'devtools.inspectedWindow'
key.substr(namespace.length + 1): 'tabId'
'tabId'.match(this.NS_RULE) is covered by the new RE but not covered by the old RE.
*/
```
In the old code `/^(.+)\.(.+)$/` is always executed against keys without dots (e.g.`'tabId'`), because namespace is never passed to `const nsProps = getAll(data.properties || {});`. As a result `matches` is always false in the old code.